Libraries/Hotcakes.PaypalWebServices/RestPaypalApi.cs

```diff
@@ -0,0 +1,288 @@
+                    {
+                        InvoiceId = invoiceId,
+                        AmountWithBreakdown = new AmountWithBreakdown()
+                        {
+                            CurrencyCode = currencyCodeType,
+                            Value = orderTotal,
+                            AmountBreakdown = new AmountBreakdown()
+                            {
+                                ItemTotal = new Money
+                                {
+                                    CurrencyCode = currencyCodeType,
+                                    Value = itemsTotal
+                                },
+                                TaxTotal = new Money
+                                {
+                                    CurrencyCode = currencyCodeType,
+                                    Value = taxTotal
+                                },
+                            }
+                        }
+                    }
+                }
+            };
+
+            // Call API with your client and get a response for your call
+            var request = new OrdersCreateRequest();
+            request.Prefer("return=representation");
+            request.RequestBody(order);
+            response = await client().Execute(request);
+
+            return response;
+        }
+
+
+        public async Task<HttpResponse> GetOrder(string orderId)
+        {
+            OrdersGetRequest request = new OrdersGetRequest(orderId);
+            var response = await client().Execute(request);
+            return response;
+        }
+
+        public async Task<HttpResponse> CaptureOrder(string orderId)
+        {
+            var request = new OrdersCaptureRequest(orderId);
+            request.Prefer("return=representation");
+            request.RequestBody(new OrderActionRequest());
+            var response = await client().Execute(request);
+            var result = response.Result<Order>();
+            
+            return response;
+        }
+
+        public async Task<HttpResponse> AuthorizeOrder(string orderId)
+        {
+            var request = new OrdersAuthorizeRequest(orderId);
+            request.Prefer("return=representation");
+            request.RequestBody(new AuthorizeRequest());
+            var response = await client().Execute(request);
+            
+            return response;
+        }
+
+        public async Task<HttpResponse> CaptureAuthorizedOrder(string AuthorizationId,string value,
+            string currencyId, string invoiceId)
+        {
+            var capture = new CaptureRequest
+            {
+                Amount = new PayPalCheckoutSdk.Payments.Money()
+                {
+                    Value = value,
+                    CurrencyCode = currencyId
+                },
+                InvoiceId = invoiceId,
+            };
+            var request = new AuthorizationsCaptureRequest(AuthorizationId);
+            request.Prefer("return=representation");
```

Comment:
Please make static strings like this a `constant`.

Reply:
OK done it.